RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -11,7 +11,7 @@
     ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
     pulseJitter           = cms.double(1.),           #GeV/bin 
     ###
-    meanTime              = cms.double(0.),   #ns 
+    meanTime              = cms.double(2.),   #ns 
```

Comment:
Was this change already discussed and well understood within HCAL DPG, more specifically, in context of its effect on the 2016 data (and possible impact on downstream, as PF and JME)?

e.g., from jenkins tests in SinglePhoton wf 136.731:
the energy of hits goes up a bit on average
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_8_1_X_2016-09-22-0900+15959/16018/validateJR/all_OldVSNew_RunSinglePh2016Bwf136p731/all_OldVSNew_RunSinglePh2016Bwf136p731c_log10HBHERecHitsSorted_hbhereco__reRECO_obj_obj_energy.png

it propagates to higher level quantities
like caloTowers
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_8_1_X_2016-09-22-0900+15959/16018/validateJR/all_OldVSNew_RunSinglePh2016Bwf136p731/all_OldVSNew_RunSinglePh2016Bwf136p731c_log10CaloTowersSorted_towerMaker__reRECO_obj_obj_energy151.png
pf neutral hadron yields are up by ~4%
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_8_1_X_2016-09-22-0900+15959/16018/validateJR/all_OldVSNew_RunSinglePh2016Bwf136p731/all_OldVSNew_RunSinglePh2016Bwf136p731c_log10recoPFCandidates_particleFlow__reRECO_obj_pt55.png

from the increase in jet yield by about 1%, I can guess there is sub-% (O(0.1%)?) level average increase in response ... unclear if it needs recalibration anywhere
https://cmssdt.cern.ch/SDT/jenkins-artifacts/baseLineComparisons/CMSSW_8_1_X_2016-09-22-0900+15959/16018/validateJR/all_OldVSNew_RunSinglePh2016Bwf136p731/all_OldVSNew_RunSinglePh2016Bwf136p731c_recoPFJets_ak4PFJets__reRECO_obj_eta.png

The change in the time distribution itself is much more obvious, but I was told in the past that it is a rather secondary variable that by itself does not contribute directly to common physics observables